specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2021-07-01/examples/Updates/Apply.json

```diff
@@ -0,0 +1,43 @@
+{
+  "parameters": {
+    "subscriptionId": "20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+    "resourceGroupName": "System.Redmond",
+    "updateLocation": "Redmond",
+    "updateName": "Microsoft1.0.180302.1",
+    "api-version": "2021-07-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/3c2a4c80-e594-46eb-b923-98fd98d50155/resourcegroups/System.Redmond/providers/Microsoft.Update.Admin/updateLocations/redmond/updates/Microsoft1.0.180410.1/updateRuns/5495b71f-692c-4580-8dcd-03dc6ce00209",
+        "name": "5495b71f-692c-4580-8dcd-03dc6ce00209",
+        "type": "Microsoft.Update.Admin/updateLocations/updates/updateRuns",
+        "location": "redmond",
+        "tags": {},
+        "properties": {
+          "progress": {
+            "name": "Downloading"
+          },
+          "duration": "PT0S",
+          "state": "InProgress"
+        }
+      }
+    },
```

Comment:
Can two responses really be present? This seems like a typo. Just the 202 should be here.

Reply:
should be only 202.